tests/cli/test_cli_install.py

```diff
@@ -15,42 +13,47 @@
 def prefix(tmpdir):
     prefix = tmpdir.mkdir("cli_install_prefix")
     test_env = tmpdir.mkdir("cli_install_test_env")
-    run_command(Commands.CREATE, str(prefix), 'python=3.10')
+    run_command(Commands.CREATE, str(prefix), "python=3.9")
```

Comment:
lowering the version?

Reply:
Yeah, with 3.10 the `test_find_conflicts_called_once` test, where this fixture is used, won't work anymore since the [`statistics` package](https://anaconda.org/main/statistics/files) used to be unavailable (what it tested for) but has been packaged for 3.10 for some reason.